README.md

```diff
@@ -1,105 +1,131 @@
-Pagila
-======
+# Pagila Docker Compose
```

Comment:
I think my auto formatter went nuts. If you do want this PR, I can clean this up, just let me know.